expo/gbe/views/review_costume_view.py

```diff
@@ -1,118 +1,56 @@
-from django.contrib.auth.decorators import login_required
-from django.core.urlresolvers import reverse
-from django.http import HttpResponseRedirect
-from django.shortcuts import (
-    render,
-    get_object_or_404,
-)
-from expo.gbe_logging import log_func
-from gbe.functions import (
-    validate_perms,
-    get_conf,
-)
 from gbe.forms import (
     CostumeBidSubmitForm,
     CostumeDetailsSubmitForm,
     ParticipantForm,
     PersonaForm,
-    BidStateChangeForm,
     BidEvaluationForm,
 )
-from gbe.models import (
-    BidEvaluation,
-    Costume,
-)
+from gbe.models import Costume
+from gbe.views import ReviewBidView
 
 
-@login_required
-@log_func
-def ReviewCostumeView(request, costume_id):
-    '''
-    Show a bid  which needs to be reviewed by the current user.
-    To show: display all information about the bid, and a standard
-    review form.
-    If user is not a reviewer, politely decline to show anything.
-    '''
-    reviewer = validate_perms(request, ('Costume Reviewers', ))
-    costume = get_object_or_404(
-        Costume,
-        id=costume_id
-    )
-    if not costume.is_current:
-        return HttpResponseRedirect(
-            reverse('costume_view', urlconf='gbe.urls', args=[costume_id]))
-    conference, old_bid = get_conf(costume)
-    costume_form = CostumeBidSubmitForm(instance=costume,
-                                        prefix='Costume Proposal')
-    details = CostumeDetailsSubmitForm(instance=costume)
+class ReviewCostumeView(ReviewBidView):
+    reviewer_permissions = ('Costume Reviewers',)
+    coordinator_permissions = ('Costume Coordinator',)
+    performer_prefix = "The Performer"
+    bidder_prefix = "The Owner"
+    bid_prefix = "The Costume"
+    bidder_form_type = PersonaForm
+    object_type = Costume
+    bid_form_type = CostumeBidSubmitForm
+    bid_view_name = "costume_view"
+    review_list_view_name = 'costume_review_list'
+    changestate_view_name = 'costume_changestate'
 
-    performer = PersonaForm(instance=costume.performer,
-                            prefix='The Performer')
+    def groundwork(self, request, args, kwargs):
+        super(ReviewCostumeView, self).groundwork(request, args, kwargs)
+        self.details = CostumeDetailsSubmitForm(instance=self.object)
+        self.performer = self.bidder_form_type(instance=self.object.performer,
+                                               prefix=self.performer_prefix)
```

Comment:
a point of clarity on this one... the bidder is actually the "owner" the "performer" is the person who originally wore the costume in a show.

Not a blocker - but this is a bit differerent that the pattern in act and class with that regards.